dockstore-webservice/src/main/java/io/dockstore/webservice/core/Validation.java

```diff
@@ -0,0 +1,158 @@
+/*
+ *    Copyright 2018 OICR
```

Comment:
We probably all need to edit our templates

Reply:
${today.year}